opm/grid/cpgrid/CpGridData.cpp

```diff
@@ -2177,15 +2279,10 @@ CpGridData::refineSingleCell(const std::array<int,3>& cells_per_dim, const int&
     if (nonRepeated_parentCorners.size() != 8){
         OPM_THROW(std::logic_error, "Cell is not a hexahedron. Cannot be refined (yet).");
     }
-    // Get ijk from parent cell (Cartesian index). Needed to compute width, length, and height of the parent cell.
-    std::array<int,3> ijk;
-    getIJK(parent_idx, ijk);
-    // Get dx,dy,dz for each cell of the patch to be refined
-    const auto& [widthX, lengthY, heightZ] = getWidthsLengthsHeights(ijk, {ijk[0]+1, ijk[1]+1, ijk[2]+1});
     // Refine parent cell
     parent_cell.refineCellifiedPatch(cells_per_dim, refined_geometries, refined_cell_to_point, refined_cell_to_face,
                                      refined_face_to_point, refined_face_to_cell, refined_face_tags, refined_face_normals,
-                                     {1,1,1}, widthX, lengthY, heightZ);
+                                     {1,1,1}, /*widthX, lengthY, heightZ*/ {1.}, {1.}, {1.});
```

Comment:
I just wonder whether the last three parameter could be computed inside `refineCellifiedPatch`, but that is not urgent,

Reply:
Oh, that reminds me that I wanted to change the name of that method since now we simple refine one cell. Maybe we postpone that since we might need in the future this more general version with width, length, and height as additional function arguments.

Reply:
Can't the width, height, length be computed from the triplet `{a, b, c}` before? `width=1./a`, etc.

Reply:
I would say we do not need any height, width, or length vectors here, since we are refining single cells, so we can just use {1., 1., 1.}. 

Heights, widths, and lengths were introduced due to the old-refinement-approach, refining a block of cells treating it as a huge single cell. The role of heights, widths, and lengths was to make sure that we have the refined children cells of each parent cell occupying the volume of their parent cell. 

When we set {1., 1., 1.}, and, let's say we want {nx, ny, nz} children cells in each direction, then the volume of each children - in the unit cube - is (1./nx)*(1./ny) *(1./nz). This is perfect for refinement of a single-cell. ( Yet not perfect for a block of at least two cells with different heights, widths, and/or lengths).